core/src/main/java/org/frankframework/scheduler/job/IbisActionJob.java

```diff
@@ -22,25 +22,28 @@
 
 import org.frankframework.configuration.AdapterManager;
 import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.configuration.ConfigurationWarning;
 import org.frankframework.core.Adapter;
 
 import org.frankframework.doc.Mandatory;
 
-import org.frankframework.management.IbisAction;
+import org.frankframework.management.Action;
 import org.frankframework.scheduler.JobDef;
 
 import org.frankframework.util.EnumUtils;
 
+@ConfigurationWarning("IbisActionJob has been renamed to ActionJob")
+@Deprecated(since = "8.2.0", forRemoval = true)
 public class IbisActionJob extends JobDef {
```

Comment:
Als deze verder het zelfde is als de nieuwe ActionJob kan je die dan niet beter extenden?